### PR TITLE
Recipe App Implementations

### PR DESCRIPTION
**Pull request by H-Ramalan**
I @H-Ramalan and my partner @HafizMassamTabraizKhan have completed working on the group project Recipe App. In the applications, we implemented these [Project Requirements](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#project-requirements) as specified.

### CODE REVIEW

File: app/views/foods/_form.html.erb
Comment (franclobo) [inline app/views/foods/_form.html.erb:1]: - [ ] There is a difference between the form and the result, please 🙏🏽 fix the differences in quantity and price for a better UX and to have the correct information 👇🏽 

| Form | Show page |
|---|---|
| image | image |

File: app/views/foods/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+<h1>Foods#show</h1>
+<section class = "">
+   <div class="">
+      <h2>Food Information</h2>
+      <p>Food Name: <strong><%= @food.name%></strong></p>
+      <p>Measurment Unit: <strong><%=  @food.measurement_unit%></strong></p>
+      <p>Quantity: <strong><%= @food.price%></strong></p>
+      <p>Unit Price: <strong><%= @food.quantity%></strong></p>
+   </div>
```
Comment (franclobo) [inline app/views/foods/show.html.erb]: - [ ] May the styles are not linked to the show page, please fix it for a better UI 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @H-Ramalan, 

#  Suggested changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

- Correct use of partials ✔️ 
- All tests passed ✔️ 
- Good UI ✔️ 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @H-Ramalan, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Requested change implemented 💯 


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**